qemu/tests/cfg/hugepage_mem_stress.cfg

```diff
@@ -3,10 +3,17 @@
     virt_test_type = qemu
     kill_vm = yes
     setup_hugepages = yes
-    extra_params += " -mem-path /mnt/kvm_hugepage"
     Linux:
         del stress_args
         stress_custom_args = "--vm %d --vm-bytes 256M --timeout 30s"
     Windows:
         install_path = "C:\Program Files\JAM Software\HeavyLoad"
```

Comment:
Would you please double check that path? It's not right for win2019 guest. I don't see heavyload running inside guest though test passes.

Reply:
Find the reason i386 uses this path, x86_64 is different.
corrected